app/static/css/style.css

```diff
@@ -115,35 +128,76 @@ footer {
 }
 
 .thin-footer {
-  font-family:'Nunito-Sans', sans-serif;
+  font-family:'Inter', sans-serif;
   font-weight: 100;
   letter-spacing: 0.5px;
   color: var(--grey-mid);
 }
 
+.expand {
+  transition: all ease 0.5s;
+}
+
+.expand:hover {
+  font-size: 101%;
+  color: var(--grid-grey);
+}
+
+
+
+/**
+ * Front page sections
+ */
+ .section {
```

Comment:
I understand .content as encompassing everything, so there would only be one .content div in a page, but multiple .section divs. Correct me if I am wrong. It also looks different right now if I use .content instead of the .section class that I created

Reply:
You can have multiple `.content` divs, they just take up 80% of their parent container. It may very well look different, in which case stick to sections.

**Note**: `<section>` is already a tag. Maybe you should put your HTML on that instead of the `.section` class? If not, let's rename it so there's no confusion.

Reply:
renamed to "strip"